src/codegen.cpp

```diff
@@ -2693,12 +2720,13 @@ static bool emit_builtin_call(jl_cgval_t *ret, jl_value_t *f, jl_value_t **args,
         }
         if (jl_subtype(ty, (jl_value_t*)jl_type_type)) {
             *ret = emit_expr(args[1], ctx);
-            Value *rt_ty = boxed(emit_expr(args[2], ctx), ctx);
+            Value *rt_ty = mark_callee_rooted(boxed(emit_expr(args[2], ctx), ctx));
```

Comment:
Same comment as above. The placement pass figures it out. The callee rooted annotation is just to annotate that the only thing you're allowed to do with it is pass it to a function that callee roots.